lucida/commandcenter/controllers/Config.py

```diff
@@ -1,5 +1,6 @@
-from Service import Service
+from Service import Service, WorkerService
```

Comment:
This file is basically a reference for developers when creating workflows. Realistically, only the Service definitions that are using in the `CLASSIFIER_DESCRIPTIONS` need to be defined. For example for the demo Config that I created I only have `WE`, `IMM`, `QA`, and `DCM` defined in `SERVICES` and the corresponding workflows in `CLASSIFIER_DESCRIPTIONS`. A better way is to have a default Config file with all possible services/workflows that can be included in a Config file (which is what we currently have) and explain in a wiki how to alter the Config file to only include the services/workflows needed.